### ISSUE TITLE
Core text renderer causing squared off window corners

### ISSUE THREAD

**Issue by peteschaffner** [alloy/macvim#66]
I recently posted [a message](https://groups.google.com/forum/?fromgroups=#!topic/vim_mac/1o_ofA_IQx0) on the MacVim discussion group about this issue, and found that this only happens with your version.

In short, I have found that using your fork with the Core Text renderer enabled causes the window to somehow draw [square corners](http://my.irong8.com/image/1U0G013c0g3M/o). This is remedied by making the scrollbars visible or using the regular NSTextView renderer.

I just thought I would mention this to you and see if you could make heads or tales of it!

**Comment by alloy** [comment]
There are even worse consequences actually. For instance, see the left sidebar in this screenshot:

![Screen Shot 2013-03-15 at 12 42 22 PM](https://f.cloud.github.com/assets/2320/263283/7e208f82-8d65-11e2-8fd3-17564fb2daca.png)

I would love to see this fixed, but I have no time at all atm to look into MacVim.

**Comment by b4winckler** [comment]
Yeah, I'm really the one who needs to look into this but it is just kind of daunting and I've just started a new job so it is hard to find the time.
